app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+class ArticlesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @articles = if user_signed_in?
+                  Article
+                    .eager_load(:user)
+                    .reorder(created_at: :desc)
+                    .all
+                else
+                  []
+                end
+    @show_article_author = true
+  end
+
+  def user_articles
```

Comment:
OK this is going wrong

If you use `/articles/453` it means that you are viewing a single article with id = 453, not articles from user with id = 453.

If you want to view some user's articles you can get `/articles?user_id=453` or `/users/453/articles`.

Also it is ok to use filters described by a word e. g. `/articles/my` is ok to show only current user's articles. Other variations are considered exotic and not usually implemented.